src/components/v1/V1Project/modals/V1ConfirmPayOwnerModal.tsx

```diff
@@ -195,43 +229,87 @@ export default function V1ConfirmPayOwnerModal({
+              />
+            </Form.Item>
+            <div
+              style={{
+                fontSize: '.8rem',
+                position: 'absolute',
+                right: 7,
+                top: 7,
+              }}
+            >
+              {
+                <Sticker
+                  style={{
+                    color: colors.text.secondary,
+                    cursor: canAddMoreStickers ? 'pointer' : 'not-allowed',
+                  }}
+                  size={20}
+                  onClick={() => {
+                    canAddMoreStickers
+                      ? setAttachStickerModalVisible(true)
+                      : undefined
+                  }}
+                />
+              }
+            </div>
+            <Form.Item name="stickerUrls">
+              <StickerSelection />
+            </Form.Item>
           </Form.Item>
 
-          <Form.Item>
-            <ImageUploader
-              text={t`Add image`}
-              onSuccess={url => {
-                if (!url) return
-                const memo = form.getFieldValue('memo') || ''
-                form.setFieldsValue({
-                  memo: memo ? memo + ' ' + url : url,
-                })
-              }}
-            />
+          <Form.Item name="uploadedImage">
+            <FormImageUploader text={t`Add image`} />
           </Form.Item>
           {hasIssuedTokens && (
-            <Form.Item label={t`Receive ERC-20`}>
-              <Space align="start">
-                <Checkbox
-                  style={{ padding: 20 }}
-                  checked={preferUnstaked}
-                  onChange={e => setPreferUnstaked(e.target.checked)}
-                />
-                <label htmlFor="preferUnstaked">
-                  <Trans>
-                    Check this to mint {tokenSymbol} ERC-20 to your wallet.
-                    Leave unchecked to have your token balance tracked by
-                    Juicebox, saving gas on this transaction. You can always
-                    claim your ERC-20 tokens later.
-                  </Trans>
-                </label>
-              </Space>
+            <Form.Item
+              name="preferUnstaked"
+              valuePropName="checked"
+              extra={
+                <Trans>
+                  Check this to mint {tokenSymbol} ERC-20 to your wallet. Leave
+                  unchecked to have your token balance tracked by Juicebox,
+                  saving gas on this transaction. You can always claim your
+                  ERC-20 tokens later.
+                </Trans>
+              }
+            >
+              <Checkbox>
+                <Trans>Receive ERC-20</Trans>
+              </Checkbox>
             </Form.Item>
           )}
         </Form>
+        <AttachStickerModal
+          visible={attachStickerModalVisible}
+          onClose={() => setAttachStickerModalVisible(false)}
+          onSelect={sticker => {
+            if (typeof window === 'undefined') {
```

Comment:
**Suggestion:** would be great to separate this code out into a `handleStickerSelect` function